packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -920,15 +920,25 @@ export const makeVirtualObjectManager = (
     const makeContext = (baseRef, state) => {
       // baseRef came from valToSlot, so must be in slotToVal
       const val = requiredValForSlot(baseRef);
+      const revoke = () => contextCache.delete(baseRef);
```

Comment:
@FUDCo do I need to register this object for some internal gc purpose? Does it being a function raise any additional issues?